recipe/meta.yaml

```
@@ -45,10 +46,7 @@ outputs:
         - cmake
         - make  # [not win]
       host:
-        # Workaround for conda solver problems for osx-64. Remove when migrating
-        # to 2.27
-        - tiledb 2.26.1  # [osx and x86]
-        - tiledb >=2.26.0,<2.27  # [not (osx and x86)]
+        - tiledb >=2.27.0,<2.28  # [not (osx and x86)]
```

```suggestion
        - tiledb >=2.27.0,<2.28
```

No selectors. Those were only for the osx-64 workaround